components/resources/library/src/androidMain/kotlin/org/jetbrains/compose/resources/AndroidContextProvider.kt

```diff
@@ -37,11 +38,34 @@ fun PreviewContextConfigurationEffect() {
     }
 }
 
+/**
+ * Sets the Android instrumented context.
+ *
+ * @param context The Android context to be set as the instrumented context.
+ *                This context will be used by a ResourceReader to read test assets.
+ *
+ * Example usage:
+ * ```
+ * fun configureTestEnvironment() {
+ *     setAndroidInstrumentedContext{
```

Comment:
The example calls the function like it takes a lambda. But the actul parameter type is just Context. Probably a typo? And can be fixed by replacing {...} with (...)